core/pages/CmsPage.js

```diff
@@ -7,7 +7,13 @@ export default {
   mixins: [Composite],
   computed: {
     pageTitle () {
-      return this.$store.state.cmsPage.current ? this.$store.state.cmsPage.current.title : ''
+      return this.$store.state.cmsPage.current ? this.$store.state.cmsPage.current.meta_title || this.$store.state.cmsPage.current.title : ''
+    },
+    pageDescription () {
+      return this.$store.state.cmsPage.current ? this.$store.state.cmsPage.current.meta_description : ''
```

Comment:
wdyt about using `lodash-es/get`? I think it's more readable
`get(this.$store.state.cmsPage.current, 'meta_description', '')`